### ISSUE TITLE
R-29's missaligned hull

### ISSUE THREAD

**Issue by Xayrlen** [FakeFishGames/Barotrauma#12893]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Even though sub editor shows a normal state of R-29 sub, during campaign I have noticed that some walls of the sub are out of place.
![1](https://github.com/Regalis11/Barotrauma/assets/86978521/46c65862-c8ca-46a3-a6a1-bd86c41e5f16)
![2](https://github.com/Regalis11/Barotrauma/assets/86978521/68bd91ae-6043-4213-8a26-33c4d5aa9ee1)
![3](https://github.com/Regalis11/Barotrauma/assets/86978521/883dbbf6-7984-41f0-9472-b2aaa13cb51d)
![4](https://github.com/Regalis11/Barotrauma/assets/86978521/c9918538-afb4-4086-9c8e-43bd9a9f5a10)
![5](https://github.com/Regalis11/Barotrauma/assets/86978521/b220ad79-7737-49b0-8620-ed37e28098d0)

### Reproduction steps

1. Start a singleplayer game;
2. Obtain R-29 sub;
3. Check out it's the exterior;

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately couldn't find a way to reproduce this. The walls on R-29 seem completely normal at my end. Are you using some kind of mods by any chance?
![image](https://github.com/Regalis11/Barotrauma/assets/14351070/30325c5d-c9a2-47fe-b0ec-900bef6114ca)

**Comment by Xayrlen** [comment]
No, this is a completely vanilla campaign, that I started the same day after Treacherous Tides update was released, so it's not an old save compatibility either.

**Comment by Xayrlen** [comment]
The problem resolved itself after transfering to a new location.

**Comment by Xayrlen** [comment]
The problem resolved itself after transfering to a new location.

**Comment by NilanthAnimosus** [comment]
We should keep this open for awhile in case it happens again and hope for a save file when it occurs as to why it happened, if you can ever get it to occur again we would be grateful if you could zip and include a save file (hopefully before another level change, such as exiting when you see it instead of continueing or docking back to outpost and thus saving again).

Curiously every instance of wall (Flipped or not) was "Shell A 4 Degrees" specifically.

I tried a new campaign then buying the submarine, upgrading the walls on a redock. then a second time where I buy the upgrades then the submarine before undocking. neither showed any issues.

**Comment by gerakis** [comment]
This happened to me too. Im including my save file (multiplayer) I was using mods but nothing related to subs.
[r29 hull misaligned.zip](https://github.com/Regalis11/Barotrauma/files/13246547/r29.hull.misaligned.zip)

**Comment by Regalis11** [comment]
No known steps to repro, closing
